data/announcements/scheduled.yaml

```diff
@@ -207,3 +207,13 @@ announcements:
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
     November 6 - November 9, 2023.
+
+- name: Legacy package repositories shutdown
+  startTime: 2023-11-27T00:00:00
+  endTime: 2024-02-01T00:00:00
+  style: "background: #d95e00"
+  title: Changes to the location of Linux packages for Kubernetes
+  message: |
+    The legacy Linux package repositories (`apt.kubernetes.io` and `yum.kubernetes.io` AKA `packages.cloud.google.com`)<br/>
+    have been frozen starting from September 13, 2023 **and are going away in January 2024**, users *must* migrate.<br/>
+    Please read our [announcement](/blog/2023/08/31/legacy-package-repository-deprecation/) for more details.
```

Comment:
If we need this, can we schedule (and announce) some brownouts, where we briefly change the DNS domains to point to a simple thing returning a 5xx response?

I'd prefer to have a whole new blog article about the shutdown _if needed_ and _if feasible_.

Reply:
The *.kubernetes.io could have brownouts but there's not a lot of time left to implement this and we already urged users to switch.

The project took too long to move off of legacy rapture.

don't really have the bandwidth for a whole new post and this has all the info and already told users they should switch.

Reply:
We also don't need the announcement if the project pivots to a static mirror but that work still isn't complete and we're cutting it terribly close at this point on warning people with a real deadline.

Reply:
I'll try to see what appetite the K8s infra SIG have for a smoother transition. I think lots of people like the idea, and only a few people would be willing to offer time to help.